### DISCUSSION TITLE
Question regarding the feed response.

### DISCUSSION THREAD

**Discussion by diwakaragg** [rometools/rome#688] [Q&A] [answered]
Hi, 

I am using this library to get the feeds from a [medium](https://medium.com/@get2parag) channel. I want to know is there any way to get this data display after each feed("3 min read") in the response?

I am using the code below to get the entries -

SyndFeed feed = new SyndFeedInput().build(new XmlReader(new URL(url)));
List<SyndEntry> entries = feed.getEntries();

**Comment by antoniosanct** [comment, marked as answer]
@diwakaragg 
Someone like this:
https://medium.com/feed/@get2parag

You can only parse information inside feed, not outside. These info are not reported at each feed item. Maybe you could contact to Medium to include it in HTML content encoded or maybe in a dc item (like relation or source).

Regards,
Antonio.
